core/README.md

```diff
@@ -3,58 +3,51 @@
 
-The IOF Core Ontology is developed and formalized as an ontology using both first-order logic and version 2 of the Web Ontology Language (OWL). The use of logic ensures that each term is defined in a way that is unambiguous to humans and can be processed by computers. All terms appearing in the ontology are reviewed and curated by a working group and consensus is reached by validating usage in the context of manufacturing domain use cases.
+The IOF Core Ontology is developed and formalized as an ontology using both first-order logic and version 2 of the Web Ontology Language (OWL). The use of logic ensures that each term is defined in a way that is unambiguous to humans and can be processed by computers. The vocabulary in Core is curated by the Core Working Group and an attempt is made to validate and ground terms utilizing use cases from industry.
 
-WARNING: This repository contains a version 1, “beta release” of the IOF Core Ontology. As such, the contents may undergo some change as we resolve outstanding issues and drive to a first stable release. 
+# Contents
 
-# Contents of this Repository
+1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examplyes, and comments, serialized in rdf/xml.
+1. meta – folder containing IOF-specific annotation properties used to define and formalize ontology terms, also serialized in rdf/xml, and imported by the core ontology. 
 
-1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examples, and comments, serialized in rdf/xml.
-1. meta – a directory containing IOF-specific annotation properties used to define and formalize ontology terms, also serialized in rdf/xml, and imported by the core ontology.
-1. LICENSE – a file containing details of the OAGi copyright statement, licensing and permissions granted for this ontology.
-1. Catalog-v001.xml – a tooling file for rapid loading of the ontology in the IOF-recommended tool for viewing and editing ontologies.
+    * AnnotationVocabulary.rdf -- this is the released version of the IOF Annotation Vocabulary, dated 2022-09-29
+1. commons-to-core-mapping – folder containing selected mid-level ontologies from the OMG Commons Library of Ontologies, addressing IOF needs in the topic area of identifiers and designators, as well as mapping files to map imported terms to IOF/BFO and align annotation property usage between IOF and OMG.  
```

Comment:
```suggestion
1. commons-to-core-mapping – folder containing selected mid-level ontologies from the OMG Commons Library of Ontologies, providing an option to the IOF and OMG users with an interoperable way to use terms in the topic area of identifiers and designators from both IOF and OMG.  
```

Reply:
I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.